core/src/test/scala/org/broadinstitute/dsde/rawls/entities/EntityServiceSpec.scala

```diff
@@ -463,4 +450,160 @@ class EntityServiceSpec
       ex.errorReport.message shouldBe "Can't find attribute name non-existent-attribute"
       ex.errorReport.statusCode shouldBe Some(StatusCodes.NotFound)
   }
+
+  // all following tests can use the same exemplar data, no need to re-create it for each unit test
```

Comment:
An overall question for this test suite -- do you want to include a unit test when the input has some  `EntityAndAttributesResult` with `refEntityRecord` as some value, and others with `None`

It seems that the unit test have either some value there or `None` for all values in a given Sequence

Reply:
I've updated one of the unit tests to use refs!